package.json

```diff
@@ -26,12 +26,14 @@
   },
   "homepage": "https://github.com/TechnologyAdvice/eslint-config-ta#readme",
   "dependencies": {
-    "babel-eslint": "^6.0.0-beta.1",
-    "eslint": "^2.2.0",
-    "eslint-config-airbnb": "^6.0.2",
-    "eslint-plugin-flow-vars": "^0.2.1",
-    "eslint-plugin-mocha": "^2.0.0",
-    "eslint-plugin-react": "^4.1.0"
+    "babel-eslint": "^7.1.1",
+    "eslint": "^3.16.1",
+    "eslint-config-airbnb": "^14.1.0",
```

Comment:
Agreed.  Their config changes far too often for comfort, and many are breaking at that.  I'd like to only have to update lint rules 1 or 2 times a year if possible.